heartbeat/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.8.3
```

Comment:
@urso I assume the heartbeat docker and compose file are not really needed for this PR?

Reply:
Depending on test environment the tests themselves run in a docker environment. I added the docker container, such that heartbeat test/build environment will be more similar to the other beats.